### PR TITLE
changes to install scripts based on FOGL-2965

### PR DESCRIPTION
**Pull request by oshadmon**
Broke-down `requirements.sh` to `requirements.sh` and `extras_install.sh`  based on [FOGL-2965](https://scaledb.atlassian.net/browse/FOGL-2965).

### CODE REVIEW

File: requirements.sh [hunk 1]
```diff
@@ -25,18 +25,9 @@ set -e
...
-
-# Linux specific prerequisites - https://www.phidgets.com/docs/OS_-_Linux
-wget https://www.phidgets.com/downloads/phidget22/libraries/linux/libphidget22.tar.gz
-tar -xzvf /tmp/wind-turbine-install/libphidget22.tar.gz
-cd /tmp/wind-turbine-install/libphidget22-*
-./configure --prefix=/ && make && sudo make install
-fn=`find -name *libphidget22.rule*`
-sudo mv ${fn} /etc/udev/rules.d.
+## extras install
+## ./extras_install
```
Comment (ashish-jabble) [inline requirements.sh]: @oshadmon Uncomment the code line and add .sh in end of statement (i.e `./extras_install.sh`); as we need this when non-package installation.

**Review by ashish-jabble — CHANGES_REQUESTED** [review]

**Review by oshadmon — COMMENTED** [review]
LGTM

**Comment by ashish-jabble** [comment]
@praveen-garg Kindly review.

**Review by praveen-garg — APPROVED** [review]
lgtm! I believe you have already executed `chmod +x extras_install.sh` Hence `./extras_install.sh`

**Review by ashish-jabble — APPROVED** [review]

**Pull request merged**